Automattic/wp-calypso
# Rename Activity Log "retention" references to "display rules," "visible days"

Fixes `1200412004370260-as-1200750011125342`.

#### Changes proposed in this Pull Request

* Rename all "retention policy" references to "display rules."
* Rename all "retention days" references to "visible days."
* Rename all "retention period" references so that they describe visibility or display instead.

#### Testing instructions

* Inspect the code thoroughly, looking for any missed references to retention or erroneous new references to display rules/visible days.
* Verify that all pages still function identically to production when the feature flag is disabled.
* Full regression testing -- all changes introduced in #54900, #55169, #55206, and #55298 should function exactly as they did prior to this PR. (See individual PRs for specific testing instructions).
* Verify you don't see any new or different console errors.
